### PR TITLE
Files should retain its folder structure

### PR DESCRIPTION
**Pull request by sveisvei**
* breaking change, remove flatten as default behaviour.

cc @ivarconr @SimenB

### CODE REVIEW

**Comment by SimenB** [comment]
LGTM! Apps should handle the path themselves.

What about providing an option though? `--flat` calls basename on the file name? No real preference either way for me

**Comment by sveisvei** [comment]
@SimenB was thinking about --flatten as well, updated PR.

File: lib/file-util.js [hunk 1]
```diff
@@ -16,9 +16,9 @@ function isDirectory (dir) {
     }
 }
 
-function getFilesToUpload (assetsFolder) {
+function getFilesToUpload (assetsFolder, flatten = false) {
```
Comment (sveisvei) [inline lib/file-util.js]: Why default to true, its weird!

**Comment by SimenB** [comment]
Also, do a `./index.js --help` and paste that into the readme 😄

**Comment by SimenB** [comment]
Perfect! Not sure if we should default to flat or not. It's the current behavior, so changing it is a breaking change. On the other hand, not flattening makes more sense IMO. @ivarconr, why did you go for flattening out the file name?

File: lib/file-util.js (hunk 1 above)
Comment (SimenB) [inline lib/file-util.js]: To avoid a breaking change.

But it's just a number, so noe real problem, I suppose

**Comment by ivarconr** [comment]
I was never my intention to flatten the file structure. My initial code did not do that, the behavior possibly change when we switched to `fs-readdir-recursive`. Because of my poor testing i must take full responsibility!

**Comment by SimenB** [comment]
Heh, so it's actually my fault. Cool 😁

**Pull request merged**